examples/FirebaseNeoPixel_ESP8266/public/index.html

```diff
@@ -16,15 +16,16 @@
 -->
 <html>
 <head>
-  <script src="https://cdn.firebase.com/js/client/2.2.1/firebase.js"></script>
+  <script src="https://www.gstatic.com/firebasejs/3.5.2/firebase.js"></script>
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.9.0/jquery.min.js"></script>
   <script src="bower_components/webcomponentsjs/webcomponents-lite.js"></script>
   <link rel="import" href="./bower_components/polymer-color-picker/polymer-color-picker.html">
 </head>
 <script>
-  // TODO: Needs security.
-  var firebaseUrl = 'https://example.firebaseio.com';
-  var firebaseSecret = secret_or_token;
+  // NOTICE: Retrieve these values from the Firebase console to run the sample.
+  var firebaseDb = 'yourproject.firebaseio.com';
```

Comment:
you should also update the README instructions:
https://github.com/googlesamples/firebase-arduino/tree/master/examples/FirebaseNeoPixel_ESP8266#configuration